assignments/function-conversion.js

```diff
@@ -1,23 +1,56 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => {
+  return `${param}`;
+}
+
+//or
+
+let anotherFunction = param => `${param}`;
+
+//correct way//////////////////////////
+let anotherFunction = (param) => param;
```

Comment:
Good, so there is no need for the string interpolators here, and it seems like you realized that and removed them in the last example.